JetUtilities/src/L2Creator.cc

```diff
@@ -43,36 +50,36 @@ L2Creator::L2Creator(CommandLine& cl) {
   mpv        = cl.getValue<bool>    ("mpv",            false);
   delphes    = cl.getValue<bool>    ("delphes",        false);
   maxFitIter = cl.getValue<int>     ("maxFitIter",        30);
+  cbPar      = cl.getValue<int>     ("cbPar", -1);
 
   if (!cl.partialCheck()) return;
   cl.print();
 }
 
 void L2Creator::resetForNextAlgorithm() {
   if(ji!=0)
-  	delete ji;
+    delete ji;
   if(odir!=0)
-  	delete odir;
+    delete odir;
   if(l3dir!=0)
-  	delete l3dir;
-	if(gl3rsp!=0)
+    delete l3dir;
+  if(gl3rsp!=0)
     delete gl3rsp;
-	if(fl3rsp!=0)
+  if(fl3rsp!=0)
     delete fl3rsp;
 
   hl_rsp.reset();
   hl_refpt.reset();
-	hl_jetpt.reset();
+  hl_jetpt.reset();
 
-	vabsrsp_eta.erase(vabsrsp_eta.begin(),vabsrsp_eta.end());
+  vabsrsp_eta.erase(vabsrsp_eta.begin(),vabsrsp_eta.end());
   vabscor_eta.erase(vabscor_eta.begin(),vabscor_eta.end());
   vrelcor_eta.erase(vrelcor_eta.begin(),vrelcor_eta.end());
 }
 
 //______________________________________________________________________________
 void L2Creator::openOutputFile() {
-  if(!outputDir.EndsWith("/")) outputDir+="/";
- 	ofile = TFile::Open(outputDir+output,"RECREATE");
+  ofile = TFile::Open(output,"RECREATE");
```

Comment:
It worked fine for me.

Le 27/04/2016 02:29, Alexx Perloff a écrit :
>
> In JetUtilities/src/L2Creator.cc 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10#discussion_r61182034>:
>
> >    vabscor_eta.erase(vabscor_eta.begin(),vabscor_eta.end());
> >    vrelcor_eta.erase(vrelcor_eta.begin(),vrelcor_eta.end());
> >  }
> >
> >  //______________________________________________________________________________
> >  void L2Creator::openOutputFile() {
> > -  if(!outputDir.EndsWith("/")) outputDir+="/";
> > - 	ofile = TFile::Open(outputDir+output,"RECREATE");
> > +  ofile = TFile::Open(output,"RECREATE");
>
> That may be, but now you are changing the functionality of a program 
> without regard for the workflow of the other users. Anyway, I didn't 
> see any movement on this, so I'm going to assume you are still 
> considering what to do.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10/files/0f73e5f4798a6aa76f70304590ae8f2aafd9668d#r61182034>
>